### DISCUSSION TITLE
From version 5.2 error for NAV - Uncaught TypeError: i.createPopper is not a function

### DISCUSSION THREAD

**Discussion by andreysneg** [twbs/bootstrap#37627] [Q&A]
From version v5.2 error in nav menu for popup submenus:
`Uncaught TypeError: i.createPopper is not a function`

With v5.1 and early version everything works well.

Try all html from documentation:  https://getbootstrap.com/docs/5.2/components/navbar/

I try old and last version popper.min.js

The loading order is correct:

```
<SCRIPT src="/assets/jquery/jquery.min.js"></SCRIPT>
<SCRIPT src="/assets/bootstrap/bootstrap.min.js"></SCRIPT>
<SCRIPT src="/assets/popper/popper.min.js"></SCRIPT>
```

**Comment by GeoSot** [comment]
Just put popper script before bootstrap

**Reply by krizza143** [reply]
Thank youuuuuuuuuuu! this worked! been trying to solve this for hours hahaha thanks!

**Reply by PedroPetrikoski** [reply]
nao funfo

**Comment by davianramsay** [comment]
If anyone haven't figured this. The dropdowns uses a 3rd Party library called popper which is not a part of the bootstrap.min.js library so you can update your second script to 
the "bootstrap.bundle.min.js / bootstrap.bundle.js" which contains popper. had this issue for weeks before I realized what was causing it. 
For EG. <SCRIPT src="/assets/bootstrap/bootstrap.bundle.js"></SCRIPT>

**Reply by bontempzadmin** [reply]
Updating the bootstrap inclusion in angular.json fixed it for me.

"scripts": [
              "node_modules/jquery/dist/jquery.min.js",
              "node_modules/bootstrap/dist/js/bootstrap.**bundle**.js"
